src/apps/pages/sections/Tabs.astro

```diff
@@ -0,0 +1,87 @@
+---
+import _ from "underscore";
+import MultiColumn from "./MultiColumn.astro";
+import clsx from "clsx";
+
+const {
+  tabs
+} = Astro.props;
+---
+
+<div class="border-b border-gray-200">
+  <nav aria-label="Tabs" class="-mb-px flex space-x-8">
+    {
+      _.map(tabs, (tab, idx) => ( 
+        <h2 
+          class={clsx(
+            'tab-label whitespace-nowrap border-b-2 px-1 py-4 text-sm font-medium',
+            { 'border-transparent text-gray-500 hover:border-secondary/60 hover:text-secondary/60 cursor-pointer': idx !== 0 },
+            { 'border-secondary text-secondary': idx === 0 }
+          )}
+          id={tab.label}
+        >
+          { tab.label }
+        </h2>
+      ))
+    }
+  </nav>
+</div>
+
+{
+  _.map(tabs, (tab, idx) => ( 
+    <div class={clsx(
+      'tab-content py-6 flex-col',
+      { 'flex': idx === 0 },
+      { 'hidden': idx !== 0 }
+    )} id={`${tab.label}--content`} >
+      {
+        _.map(tab.content, (section) => ( 
+          <MultiColumn
+            textAlignment={section.text_alignment}
+            content={section.columns}
+            gap={section.gap}
+          />
+        ))
+      }
+    </div>
+  ))
+}
+
+<script is:inline>
+  window.document.addEventListener('DOMContentLoaded', () => {
```

Comment:
What are you thoughts on using a React component with `client:load` vs. inline Javascript? Is the performance savings worth the readability/maintainability penalty?